cmd/cmd.go

```diff
@@ -47,6 +47,19 @@ const oldCore = "/snap/ubuntu-core/current"
 // old ubuntu-core snaps older than this aren't suitable targets for re-execage
 const minOldRevno = 126
 
+func shouldReexec(envKey string) bool {
```

Comment:
IMHO the function should probe the environment internally and just have a signature `func shouldReExec() bool`  but feel free to do this later.